{"url":"https://api.github.com/repos/owncloud/core/issues/6536","labels_url":"https://api.github.com/repos/owncloud/core/issues/6536/labels{/name}","comments_url":"https://api.github.com/repos/owncloud/core/issues/6536/comments","events_url":"https://api.github.com/repos/owncloud/core/issues/6536/events","html_url":"https://github.com/owncloud/core/issues/6536","id":24628732,"number":6536,"title":"database is locked error","user":{"login":"netbulae","id":1678578,"avatar_url":"https://gravatar.com/avatar/5df4d6913a1aa28506196aaf85aa2d3b?d=https%3a%2f%2fidenticons.github.com%2fd8027a80fbf0e1a34bdf7de3a06c0546.png&r=x","gravatar_id":"5df4d6913a1aa28506196aaf85aa2d3b","url":"https://api.github.com/users/netbulae","html_url":"https://github.com/netbulae","followers_url":"https://api.github.com/users/netbulae/followers","following_url":"https://api.github.com/users/netbulae/following{/other_user}","gists_url":"https://api.github.com/users/netbulae/gists{/gist_id}","starred_url":"https://api.github.com/users/netbulae/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/netbulae/subscriptions","organizations_url":"https://api.github.com/users/netbulae/orgs","repos_url":"https://api.github.com/users/netbulae/repos","events_url":"https://api.github.com/users/netbulae/events{/privacy}","received_events_url":"https://api.github.com/users/netbulae/received_events","type":"user","site_admin":false},"labels":[{"url":"https://api.github.com/repos/owncloud/core/labels/db%3amysql","name":"db:mysql","color":"444444"}],"state":"open","assignee":null,"milestone":null,"comments":2,"created_at":"2013-12-20t15:17:00z","updated_at":"2013-12-23t10:42:34z","closed_at":null,"pull_request":{"html_url":null,"diff_url":null,"patch_url":null},"body":"### expected behaviour\r\nmy admin changed the password of my account, i tried logging in directly after he told me the new password\r\n\r\n### actual behaviour\r\nan exception occurred while executing 'insert into oc_preferences (userid, appid, configkey, configvalue) values (?, ?, ?, ?)' with params [\"username\", \"files\", \"cache_version\", 5]: sqlstate[hy000]: general error: 5 database is locked\r\n\r\n\r\n### steps to reproduce\r\n1. change user password\r\n2. try to login directly with that account\r\n3. error on screen still logged in\r\n\r\n### server configuration\r\noperating system:\r\ncentos 6.5\r\n\r\nweb server: \r\napache\r\ndatabase:\r\nmysql\r\n\r\nowncloud version: (see owncloud admin page)\r\n6.0.0a (stable) \r\n\r\nupdated from an older owncloud or fresh install:\r\nfresh install\r\n\r\nare you using external storage, if yes which one:\r\nno\r\n\r\n### client configuration\r\nbrowser:\r\nfirefox 22.0\r\noperating system:\r\nfedora 17\r\n","closed_by":null}